packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -280,18 +280,29 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>> extend
         if (this.suggestionsControl && this.suggestionsControl.handleKeyDown(keyCode)) {
           ev.preventDefault();
           ev.stopPropagation();
+          this._updateActiveDescendant();
         }
         break;
 
       case KeyCodes.down:
         if (this.suggestionsControl && this.suggestionsControl.handleKeyDown(keyCode)) {
           ev.preventDefault();
           ev.stopPropagation();
+          this._updateActiveDescendant();
         }
         break;
     }
   };
 
+  private _updateActiveDescendant(): void {
+    if (this.props.inputElement && this.suggestionsControl && this.suggestionsControl.selectedElement) {
+      const selectedElId = this.suggestionsControl.selectedElement.getAttribute('id');
+      if (selectedElId) {
+        this.props.inputElement.setAttribute('aria-activedescendant', selectedElId as string);
```

Comment:
This feels a bit dangerous. I know that we add an event listener to the input element, but this could override existing active descendants that already exist on the input element. @amyngu do you have any ideas on being able to pass the active descendant to the input control?

Reply:
You might also run into some weird timing issues with this, since this will be synchronous, but updating the selected item will be asynchronous thanks to setstate.

Reply:
This existing one does not work. It is not rendered when user is pressing down arrow, so the activedescendant is not updated when user press the down arrow. I m assume from a accessibility perspective the down arrow should trigger the update on activedescendant.

Reply:
Unfortunately I can't think of a better way to update the input :/